pkg/aggregator/sketch_map.go

```diff
@@ -35,12 +35,16 @@ func (m sketchMap) insert(ts int64, ck ckey.ContextKey, v float64) bool {
 	return true
 }
 
-func (m sketchMap) insertN(ts int64, ck ckey.ContextKey, v float64, n uint) bool {
-	if math.IsInf(v, 0) || math.IsNaN(v) {
+func (m sketchMap) insertInterp(ts int64, ck ckey.ContextKey, l float64, u float64, n uint) bool {
```

Comment:
Could we add comments or more explicit variable names on this function? (especially for`l`ower & `u`pper)